src/Type/PostType/PostTypeType.php

```diff
@@ -42,7 +43,9 @@ public function __construct() {
 		$config = [
 			'name' => self::$type_name,
 			'description' => __( 'An Post Type object', 'wp-graphql' ),
-			'fields' => self::fields(),
+			'fields' => function() {
```

Comment:
Do we have to make these changes other places as well? Seems like it would have to happen here too -> https://github.com/wp-graphql/wp-graphql/blob/master/src/Type/TermObject/TermObjectType.php#L56